src/components/tasks/card/index.tsx

```diff
@@ -106,15 +108,22 @@ const Card: FC<Props> = ({
             height={iconHeight}
           />
           <span className={classNames.cardSpecialFont}>Due Date</span>
-          <span
-            className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'endsOn')}
-            role="button"
-            tabIndex={0}
-          >
-            {fromNowEndsOn}
-          </span>
+          {shouldEdit ? (
```

Comment:
When does shouldEdit become true?

Reply:
when user is Authorized and int url edit=true
both will true then shouldEdit bexomes true

Reply:
is any user allowed to do it or just super user/admin